### PR TITLE
Question page

### PR DESCRIPTION
**Pull request by partha8**
Added the question page, please share your views. Thank you!

Here's the preview: https://deploy-preview-3--algern-quiz.netlify.app/components/question-page.html

### CODE REVIEW

File: components/question-page.html [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    <link rel="stylesheet" href="../styles/style.css">
+    <link rel="stylesheet" href="../styles/question-page.css">
+
+    <title>Algern Quiz</title>
+</head>
+
+<body>
+    <header class="header">
+        <nav class="navbar">
+            <a href="../index.html">
```
Comment (mikesch803) [inline components/question-page.html]: you can use / only to navigate to the index.html

**Review by mikesch803 — APPROVED** [review]
Everything else looks good

**Review by anik31 — APPROVED** [review]
Everything looks good. Suggestion - there is so much space on top & bottom of the question, you can stack the options & they can take full width. I think this will look good, not sure but you can try it out later.

File: components/question-page.html (hunk 1 above)
Comment (partha8) [reply components/question-page.html]: done

**Comment by partha8** [comment]
> Everything looks good. Suggestion - there is so much space on top & bottom of the question, you can stack the options & they can take full width. I think this will look good, not sure but you can try it out later.

It won't look good, and plus the space will be adjusted when you decrease the screen size. Thank you for reviewing.

**Pull request merged**